### PR TITLE
update version

### PR DESCRIPTION
**Pull request by mina-sealink**
- Update the readme file
- Change the Tooltip's position from right to bottom and add styles for it
- Update the version

### CODE REVIEW

**Review by mnoack — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
-# sealink-ecom-engine-css
+# Sealink Ecom Engine CSS
+
+SeaLink Ecom Engine shared styles
+
+
+### Using
```
Comment (dwaring) [inline README.md]: Usage

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+
+### Usage
+
+    gem install sealink-ecom-engine-css --save
+
+
+### Import the style file to your project main scss/sass
+
+    @import "sealink-ecom-engine-css/sass/ecom"
```
Comment (dwaring) [inline README.md]: Should we rename the stylesheet to something like `ecom-engine-styles`?

File: README.md (hunk 1 above)
Comment (mina-sealink) [reply README.md]: changed it!

File: README.md (hunk 2 above)
Comment (mina-sealink) [reply README.md]: changed the file name

**Review by dwaring — APPROVED** [review]

**Pull request merged**